ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -584,6 +584,16 @@ export default class TaskGroup extends Component {
     }
   };
 
+  shouldIgnoreError(error) {
+    if (!error) {
+      return true;
+    }
+
+    // Outside Firefox CI, quite often we can have a
+    // task group without a decision task.
```

Comment:
Maybe "Task groups do not necessarily have a decision task, so handle task-not-found errors gracefully"?